Description: Fixes to allow midgard r16p0 to build against 4.14 headers
 Accommodate user_buf_map API change
Author: Guillaume Tucker <guillaume.tucker@collabora.com>
Forwarded: no
Reviewed-By: Wookey <wookey@debian.org>
Last-Update: 2018-01-15

--- a/mali_kbase_mem.c
+++ b/mali_kbase_mem.c
@@ -2159,7 +2160,7 @@ static int kbase_jd_user_buf_map(struct
 			address,
 			alloc->imported.user_buf.nr_pages,
 			reg->flags & KBASE_REG_GPU_WR ? FOLL_WRITE : 0,
-			pages, NULL);
+			pages, NULL, NULL);
 #endif
 
 	if (pinned_pages <= 0)
